### DISCUSSION TITLE
found Error when scrolling or zooming in PaddleocrLabel.py

### DISCUSSION THREAD

**Discussion by Gopipatwa** [PaddlePaddle/PaddleOCR#8015] [Show and tell]
Traceback (most recent call last):
  File "C:\Projects\paddleocrproject\PaddleOCR\PPOCRLabel\PPOCRLabel.py", line 1475, in zoomRequest
    self.addZoom(scale * units)
  File "C:\Projects\paddleocrproject\PaddleOCR\PPOCRLabel\PPOCRLabel.py", line 1434, in addZoom
    self.setZoom(self.zoomWidget.value() + increment)
  File "C:\Projects\paddleocrproject\PaddleOCR\PPOCRLabel\PPOCRLabel.py", line 1431, in setZoom
    self.zoomWidget.setValue(value)
TypeError: setValue(self, int): argument 1 has unexpected type 'float'

**Comment by Evezerest** [comment]
Can you describe how this bug was triggered? and your OS, the version of PPOCRLabel and paddleocr?

**Reply by Gopipatwa** [reply]
latest version of paddleocr
when i started labelling my data with ppocrlabel.py.
After loading img dir and zoom image then this error throw

**Reply by TasneemVKhan** [reply]
> Can you describe how this bug was triggered? and your OS, the version of PPOCRLabel and paddleocr?

I have the same error, can you please help?

**Reply by Evezerest** [reply]
is your OS Mac or Windows?  Have you tried the latest version 2.1.3 of PPOCRLabel?

**Reply by TasneemVKhan** [reply]
OS Windows 11 

How do I check the version of PPOCRLabel? 
From where will get the latest version of PPOCRLabel ? 

Also please can you provide steps how to train our custom dataset for table recognition ?

**Reply by SplyzerRB** [reply]
Same error + more : https://github.com/PaddlePaddle/PaddleOCR/discussions/9127 any fix @Evezerest?

**Reply by SplyzerRB** [reply]
@TasneemVKhan @Gopipatwa fixed it by downgrading to python 3.9.10
